lib/improver/tests/ensemble_calibration/ensemble_calibration/test_EnsembleCalibration.py

```diff
@@ -72,152 +71,34 @@
-            _create_truth(self.current_temperature_forecast_cube))
-
-        self.expected_temperature_predictor_data = np.array(
-            [[273.15, 274.15, 275.15],
-             [276.15, 277.15, 278.15],
-             [279.15, 280.15, 281.15]], dtype=np.float32)
-
-        self.expected_temperature_variance_data = np.array(
-            [[0., 0., 0.],
-             [0., 0., 0.],
-             [0., 0., 0.]])
-
-        # Create a cube for testing wind speed.
-        data = np.array([[[1., 2., 3.],
-                          [4., 5., 6.],
-                          [7., 8., 9.]],
-                         [[1., 2., 3],
-                          [4., 5., 6.],
-                          [7., 8., 9.]],
-                         [[1., 2., 3.],
-                          [4., 5., 6.],
-                          [7., 8., 9.]]])
-        data = data.astype(np.float32)
-        self.current_wind_speed_forecast_cube = set_up_variable_cube(
-            data, name="wind_speed", units="m s-1", realizations=[0, 1, 2])
-
-        self.historic_wind_speed_forecast_cube = (
-            _create_historic_forecasts(self.current_wind_speed_forecast_cube))
-
-        self.wind_speed_truth_cube = (
-            _create_truth(self.current_wind_speed_forecast_cube))
-
-        self.expected_wind_speed_predictor_data = np.array(
-            [[0., 1., 2.],
-             [3., 4., 5.],
-             [6., 7., 8.]], dtype=np.float32)
-
-        self.expected_wind_speed_variance_data = np.array(
-            [[0., 0., 0.],
-             [0., 0., 0.],
-             [0., 0., 0.]],
-            dtype=np.float32
-        )
-
-
-class SetupCubesWithVariance(IrisTest):
-
-    """Set up cubes for class."""
-
-    @ManageWarnings(
-        ignored_messages=IGNORED_MESSAGES, warning_types=WARNING_TYPES)
-    def setUp(self):
-        """Set up temperature and wind speed cubes for testing."""
-        # Note: test_temperature_realizations_data_check produces ~0.5K
-        # different results when the temperature forecast cube is float32
-        # below. A bug?
-        self.calibration_method = "ensemble model output_statistics"
-        data = np.array([[[0., 1., 2.],
-                          [3., 4., 5.],
-                          [6., 7., 8.]],
-                         [[1., 2., 3],
-                          [4., 5., 6.],
-                          [7., 8., 9.]],
-                         [[2., 3., 4.],
-                          [5., 6., 7.],
-                          [8., 9., 10.]]])
-        data = data + 273.15
-        data = data.astype(np.float32)
-        self.current_temperature_forecast_cube = set_up_variable_cube(
-            data, units="Kelvin", realizations=[0, 1, 2])
-
-        self.historic_temperature_forecast_cube = (
-            _create_historic_forecasts(self.current_temperature_forecast_cube))
-
-        self.temperature_truth_cube = (
-            _create_truth(self.current_temperature_forecast_cube))
-
+        super().setUp()
```

Comment:
I'm struggling a bit with this one.  Is this setting up the expected outputs?  Could you clarify the docstring and use plain English for the variable names please.